examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,362 @@
+"""
+Plotting datetime charts
+========================
+
+Plotting datetime data points is handled by :meth:`pygmt.Figure.basemap`.
+"""
+# sphinx_gallery_thumbnail_number = 0
+
+import datetime
+
+import numpy as np
+import pandas as pd
+import pygmt
+import xarray as xr
+
+########################################################################################
+# Datetime Input Types
+# --------------------
+#
+# PyGMT accepts a variety of datetime objects to plot data and create charts.
+# Aside from the built-in Python datetime object, PyGMT supports input using
+# ``pandas``, ``numpy``, ``xarray`` as well as properly formatted strings.
+# These data types can be used to plot specific points as well as get
+# passed into the ``region`` parameter to create a range of the data on an axis.
+#
+# The following examples will demonstrate how to create plots
+# using the different datetime objects.
+
+###############################################################################
+# Using Python's ``datetime``
+# ---------------------------
+#
+# In this example, Python's built-in ``datetime`` module is used
+# to create data points stored in list ``x``. The format of the
+# data is ``(Y, M, D)``. Additionally, dates are passed into the
```

Comment:
I remove this sentence because `datetime` is Python's built-in module and `datetime.date` in the code is correctly linked to the official documentation. 
```suggestion
# to create data points stored in list ``x``. Additionally, dates are passed into the
```